cli/utilities/packages.py

```diff
@@ -83,6 +83,26 @@ def install(self, pkg, nogpgcheck=False, env_vars={}):
         elif out:
             raise PackageError(f"Failed to install package '{pkg}'")
 
+    def remove(self, pkg, nogpgcheck=False, env_vars={}):
+        """Remove a package or packages
+
+        Args:
+            pkg (str): package needs to be removed
```

Comment:
One suggestion here, isn't it better to have the packages as a list? Else, the user will have to send the string like "Pkg_a <space> Pkg_b <space> Pkg_n"

Instead, accepting it as a list and then appending to command would be a good option. 
Just a suggestion, please ignore if you feel otherwise

Reply:
Thanks for the suggestion, @pranavprakash20! The method is meant to remove one package at a time.

@vamahaja What do you think?

Reply:
Looks good idea, yum supports multiple package removal at once, better to support it.